Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -46,6 +50,45 @@ public DriveSubsystem(CommonLibFactory factory, XPropertyManager propManager) {
         masterTalons = new HashMap<XCANTalon, BaseDriveSubsystem.MotionRegistration>();
         masterTalons.put(leftMaster, new MotionRegistration(0, 1, -1));
         masterTalons.put(rightMaster, new MotionRegistration(0, 1, 1));
+        
+        
+        leftMaster.createTelemetryProperties("LeftDriveMaster");
+        rightMaster.createTelemetryProperties("RightDriveMaster");
+        
+        leftTicksPerFiveFeet = propManager.createPersistentProperty("leftDriveTicksPer5Feet", 0);
+        rightTicksPerFiveFeet = propManager.createPersistentProperty("rightDriveTicksPer5Feet", 0);
+        
+    }
+    
+    /**
+     * Helper function to get the "typical" number of inches for a given number of ticks. This assumes
+     * that there isn't much difference between left/right drive encoders.
+     * @param ticks Number of encoder ticks
+     * @return Number of inches
+     */
+    public double ticksToInches(double ticks) {
+        double ticksPerInch = getAverageTicksPerInch();
+        
+        if (ticksPerInch != 0) {
```

Comment:
nit: it can be a bit easier to skim if you put error conditions first and abort if the input meets it (i.e., `if (ticksPerInch == 0) return`).

Reply:
Yes, good pattern.